.github/workflows/node-integrate.yml

```diff
@@ -143,6 +147,12 @@ jobs:
           cd node/packages/aws-lambda-sdk
           npm update --no-save
 
+      - name: Install test/go/dev-mode dependencies
```

Comment:
There shold be no dependencies installed in this folder (everything we need is installed at `node` and if something is missing let's simply add it there